packages/ui/src/components/QueryBuilder/QueryPills/FieldQueryPill.tsx

```diff
@@ -40,14 +41,8 @@ const Operator: React.FC<IOperatorProps> = ({ className = '', type }) => {
     }
 };
 
-const QueryPill: React.FC<IQueryBarProps> = ({
-    query,
-    dictionary = {},
-    showLabels,
-    onRemove,
-    currentSelectedQuery,
-}) => {
-    const containerClassNames = cx(styles.container, { [styles.selected]: currentSelectedQuery });
+const FieldQueryPill: React.FC<IFieldQueryPillProps> = ({ query, dictionary = {}, showLabels, onRemove, isBarActive }) => {
+    const containerClassNames = cx(styles.container, { [styles.selected]: isBarActive });
     return (
```

Comment:
I would save the return here, but maybe it's juste personnal opinion, do what you please!